AutonomousDemo/VehicleMotion.cs

```diff
@@ -19,7 +19,7 @@ public VehicleMotion(double currentSpeed)
             this.CurrentSpeed = currentSpeed;
         }
 
-        public Velocity PressGasPedal(double meters, int seconds)
+        public Velocity Drive(double meters, int seconds)
```

Comment:
I guess I've been stuck thinking terms of what a physical car does. 

This function calculates current velocity with respect to acceleration.